js/script.js

```diff
@@ -78,23 +83,60 @@ function draw() {
     // amount to move
     pfX += dx;
 
-    drawKirby();
+    notCollided();
 
-    if (itemLanded && notCollided()) drawItem("tomato");
+    if (itemLanded) drawItem("tomato");
+    drawKirby();
 }
 
 function drawKirby() {
-    ctx.drawImage(wkirby, -50, 190, wkirby.width * 4, wkirby.height * 4);
+    ctx.drawImage(wkirby, 20, 290, wkirby.width * 4, wkirby.height * 4);
     ctx.drawImage(ckirby, canvas.width / 2 - ckirby.width, 200, ckirby.width * 2, ckirby.height * 2);
     if (delayCount % delay == 0) {
-        wkirby.src = "assets/walkkirby/wkirby" + f + ".gif";
+        wkirby.src = walkArray[f];
         ckirby.src = "assets/Kirby_Animation_Frames/chef_kirby/ckirby" + f + ".gif";
         f++;
     }
 
-    if (delayCount < delay * 16) delayCount++;
+    if (delayCount < delay * walkArray.length) delayCount++;
     else delayCount = 1;
-    if (f > 15) f = 0;
+    // Last animation frame
+    if (f > walkArray.length - 1) {
+        if (isSuck) {
+            console.log("isSuck = false");
+
+            loadArray("default");
+            isSuck = false;
+            console.log("f1: " + f);
+        }
+        f = 0;
+    }
+
+}
+
+function loadArray(kirby) {
```

Comment:
How do you feel about loading these arrays in a diff file and just including/importing that file in this file? If we write the loading into a function there's a chance it'll execute more than once, which it shouldn't have to since these images are static.